### PR TITLE
simplify workflow

### PR DESCRIPTION
**Pull request by wullub**

### CODE REVIEW

File: .github/workflows/terraform.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  ARM_CLIENT_SECRET_DEV:    ${{ secrets.DIGITAL_STUDIO_INFRA_DEVTEST_CLIENT_SECRET }}
+  ARM_CLIENT_SECRET_PROD:   ${{ secrets.DIGITAL_STUDIO_INFRA_PROD_CLIENT_SECRET }}
+  ARM_SUBSCRIPTION_ID_DEV:  c27cfedb-f5e9-45e6-9642-0fad1a5c94e7
+  ARM_SUBSCRIPTION_ID_PROD: a5ddf257-3b21-4ba9-a28c-ab30f751b383
+  ARM_TENANT_ID:            747381f4-e81f-4a43-bf68-ced6a1e14edf
+
+jobs:
+  verify-terraform-config:
+    runs-on: ubuntu-latest
+    environment: ${{ env.ENVIRONMENT }}
```
Comment (reallydontask) [inline .github/workflows/terraform.yml:23]: I thought you said that this needed to be hardcoded or am i imagining things?

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    steps:
+      - name: testing
+        run: echo "${{ github }}"
```
Comment (reallydontask) [inline .github/workflows/terraform.yml:27]: This repository is public, I'm not 100% that you should be echoing this, lest credentials leak

File: .github/workflows/terraform.yml (hunk 1 above)
Comment (wullub) [reply .github/workflows/terraform.yml:23]: i did. I'll be testing if i am wrong

Comment (wullub) [reply .github/workflows/terraform.yml:23]: i'll do cleanup as required

File: .github/workflows/terraform.yml (hunk 2 above)
Comment (wullub) [reply .github/workflows/terraform.yml:27]: fair enough

**Pull request closed without merging**